packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -43,7 +43,7 @@ import {
 import { componentOnReady } from "../../utils/component";
 import { SLOTS as PANEL_SLOTS } from "../panel/resources";
 import { HeadingLevel } from "../functional/Heading";
-import { OverlayPositioning } from "../../components";
+import type { OverlayPositioning } from "../../utils/floating-ui";
```

Comment:
since components.d.ts will be deleted, I'm changing this to import from the actual origin